File: namsor/csharp/src/OSEG.NamsorExamples/JapaneseNameKanjiCandidates1Example.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using App.Namsor.Api;
using App.Namsor.Client;
using App.Namsor.Model;

namespace OSEG.NamsorExamples;

public class JapaneseNameKanjiCandidates1Example
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("X-API-KEY", "YOUR_API_KEY");

        try
        {
            var response = new JapaneseApi(config).JapaneseNameKanjiCandidates1(
                japaneseSurnameLatin: "Sanae",
                japaneseGivenNameLatin: "Yamamoto"
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling JapaneseApi#JapaneseNameKanjiCandidates1: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: namsor/typescript-axios/src/JapaneseNameKanjiCandidates1Example.ts
import axios, { AxiosError } from "axios";
import * as api from "namsor_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.JapaneseApi(configuration).japaneseNameKanjiCandidates1(
  "Sanae", // japaneseSurnameLatin
  "Yamamoto", // japaneseGivenNameLatin
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling JapaneseApi#japaneseNameKanjiCandidates1:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
